src/lib/components/toggle/QSwitch.scss

```diff
@@ -0,0 +1,267 @@
+@use "sass:list";
+@use "$css/mixins";
+
+$root: ".q-switch";
+$track: "#{$root}__track";
+$handle: "#{$root}__handle";
+$icon: "#{$root}__icon";
+
+$checked: "[aria-checked]";
+$unchecked: ":not([aria-checked])";
+
+$check-map: (
+  "checked": "[aria-checked]",
+  "unchecked": ":not([aria-checked])",
+);
+
+@function getColor($el, $checked: false, $hovered: null) {
+  @if $el == "track" {
+    @return if($checked, var(--primary), var(--surface-container-highest));
+  } @else if $el == "icon" {
```

Comment:
Can't we remove the `@else` if we're returning in the `@if` anyway?

Reply:
I guess we can, I'm not familiar enough with SASS to say

Reply:
5d6f37c63a4ad9b8185d98a6720880e576ed7189